### ISSUE TITLE
ARPL 106 total crash 

### ISSUE THREAD

**Issue by nemesis122** [fbelavenuto/arpl-modules#94]
[problem arpl106.txt](https://github.com/fbelavenuto/arpl/files/10156220/problem.arpl106.txt)

**Comment by nemesis122** [comment]
Hi Sorry to say it is not working updated the usb stick to 1.08 chosse 3622xs updated all modules and also tried without any module .
[1.08 with all Modules not working.txt](https://github.com/fbelavenuto/arpl-modules/files/10214592/1.08.with.all.Modules.not.working.txt)


Second created a new usb stick with 1.08 without modules with modules etc this ise is happen patching the zimage 
![error creating the loader 3622xs](https://user-images.githubusercontent.com/12814981/207233039-2fc253e1-7882-4689-8f45-950cc354602c.png)

with 1.03b all is working perfect 

thank you very much and have a nice day

**Comment by nemesis122** [comment]
will test some other things today evening also try with my mellanox 10 Gbe SFP+ card

**Comment by nemesis122** [comment]
I think here is the reason  [   12.790287] sd 6:0:0:0: [synoboot] Attached SCSI removable disk
udevadm settle failed
[   56.685321] NMI watchdog: BUG: soft lockup - CPU#2 stuck for 41s! [scsi_id:4844]
[   56.720695] Modules linked in: usb_storage ixgbe(O)

so in this case again the ixgbe driver is broken when its not taken from the source as we have in 1.03b
i
could we add this module as we have  in 1.03b because this version from arpl 1.03 is working great 
thank you very much 
see ya
Michael

**Comment by nemesis122** [comment]
i think this is also releated to this 
https://github.com/pocopico/rp-ext/issues/103
the ixgbe driver is included in DSM and vanilla and causes the problem.
Anyway with 1.03 all was perfekt what is different for the ixgbe driver from 1.03b to 1.04- 1.08b ??
thank you
Michael

**Comment by nemesis122** [comment]
So i think i can disable the PCI device intel 520 in the bios and create the loader with arpl 1.08 and after them enabled it and change the mac adress and it should work :-)

But with 1.03b and 0.5 arpl  it is working great why?? what is different with the module ixgbe in 1.03b / 0.5 /  and others like 1.04 - 1.08b etc ??????

**Comment by fbelavenuto** [comment]
> Second created a new usb stick with 1.08 without modules with modules etc this ise is happen patching the zimage

Okay, I don't know what could have happened here!

**Comment by fbelavenuto** [comment]
> But with 1.03b and 0.5 arpl it is working great why?? what is different with the module ixgbe in 1.03b / 0.5 / and others like 1.04 - 1.08b etc ??????

I don't know!! I took the same modules from version 1.03 and put them in version 1.07 and it didn't work!
If you can test it, get the files inside /mnt/p3/modules from the 1.03 image and put it on your current USB flash drive, rebuild the loader and test it.

**Comment by nemesis122** [comment]
Hi thank you for your feedback in 103b the ixgbe driver was used from the source (<--this means DSM? ) maybe there is en other loader config ? or the module use has changed?
ty

**Comment by fbelavenuto** [comment]
Now I have removed the ixgbe from all the models that already have it! 
I'm going to look for a network card that uses this driver for further testing.
Thanks.

**Comment by fbelavenuto** [comment]
I use a Xeon server with proxmox, so most of the time I use virtio_net. I have access to some machines at work so I test with some Realtek models and Intel i211 network cards (igb driver).

**Comment by nemesis122** [comment]
Por lo tanto, está trabajando con el cargador ARPL 1.09 creado con estos pasos:
1 Actualizar ARPL a 1.09 
2 Actualizar todos los módulos y complementos
3 Reinicio
4 crear 3622 xs con todos los módulos incluidos 
5 esta versión del cargador 1.09 viene im mi caja fuerte :-)
6 ¿Cuál fue el problema? 
muchas gracias Maestro 


" So it is working with the ARPL 1.09 Loader Createdt with this steps:
1 Update ARPL to 1.09 
2 Update all Modules and Addons
3 Reboot
4 create 3622 xs with all Modules included 
5 this Loader Version 1.09 comes im my safe :-)
6 what was the problem? 
thank you very much  Master "
[ARPL 1.09 all Modules included version 3622xs 42962 working like a dream.txt](https://github.com/fbelavenuto/arpl-modules/files/10221289/ARPL.1.09.all.Modules.included.version.3622xs.42962.working.like.a.dream.txt)

**Comment by nemesis122** [comment]
this drives me crazy second time creating the loader again and now kernel panic 
:-/

**Comment by nemesis122** [comment]
So i found it but i dont know why 
so first create the loader install DSM all is fine create the storage or etc in DSM all is fine ...
reboot the server and kernel panic 
[second reboot kernel panic.txt](https://github.com/fbelavenuto/arpl-modules/files/10221637/second.reboot.kernel.panic.txt)

**Comment by nemesis122** [comment]
This is the strange behavior at the second boot 
[ 127.490716] <redpill/bios_shim.c:100> broadwellnk_synobios BIOS went away - you may get a kernel panic if YOU unloaded it

**Comment by nemesis122** [comment]
with 1.03b all is working great tested again maybe this kernel panic is not related to the ixgbe ???
ev. to en other issue that is changed from 1.03b ..... to 1.09b in the loader because unload the bios is strange 
[ 127.490716] <redpill/bios_shim.c:100> broadwellnk_synobios BIOS went away - you may get a kernel panic if YOU unloaded it

**Comment by nemesis122** [comment]
i will test with the mellanox card an let you know

**Comment by nemesis122** [comment]
I have now tested with arpl 1.03b but all addons and Modules from the version 1.08b but the loader version is still 1.03b and the server crashed pls find the log attached
[1.03b with updated Modules abd Addons to 1.08 but arpl is still 1.03b crash.txt](https://github.com/fbelavenuto/arpl-modules/files/10224247/1.03b.with.updated.Modules.abd.Addons.to.1.08.but.arpl.is.still.1.03b.crash.txt)

**Comment by fbelavenuto** [comment]
Hi nemesis122, this last log do not have a kernel panic!
Please do some tests, uses the v1.0-beta3 and update only modules, build and test

**Comment by nemesis122** [comment]
this is  already done this morning and it is not working (1.03b with updates module )but with 920+ it is working

**Comment by nemesis122** [comment]
Please find attached the logs 
workflow:
update only the modules 
Select 3622xs Version 42962 
Deselect all Modules
Build the Loader 
 and Booting --> Crash

Second 
Reboot edit the Loader 
Add all Modules 
Build the loader 
Booting 
--> 
[3622xs 1.03b only update the Modules and booting without modules_error synboot removed in the log.txt](https://github.com/fbelavenuto/arpl-modules/files/10230381/3622xs.1.03b.only.update.the.Modules.and.booting.without.modules_error.synboot.removed.in.the.log.txt)
[3622xs 1.03b only update the Modules and booting with all modules.txt](https://github.com/fbelavenuto/arpl-modules/files/10230382/3622xs.1.03b.only.update.the.Modules.and.booting.with.all.modules.txt)
Crash

**Comment by fbelavenuto** [comment]
The error is this:
```
[   11.766378] sd 6:0:0:0: [synoboot] Attached SCSI removable disk
udevadm settle failed
[   56.122881] NMI watchdog: BUG: soft lockup - CPU#3 stuck for 41s! [scsi_id:4909]

```
`[   52.119368] NMI watchdog: BUG: soft lockup - CPU#3 stuck for 41s! [scsi_id:4830]`
Some trouble with SCSI!!

Try one thing, go to advanced menu, enable "direct boot" and boot

**Comment by nemesis122** [comment]
with direct boot it is working but why ?

**Comment by nemesis122** [comment]
920+ is working great is there 
disadvantage to 3622xs
?

**Comment by fbelavenuto** [comment]
> 920+ is working great is there disadvantage to 3622xs ?

No, the 920 is device-tree, it only works with native SATA. If it's working fine for you I recommend using the 920.

**Comment by nemesis122** [comment]
it is realy strange with this loader since 1.06
i have changed the network adapater to Mellanox 3 and it is crash again
install all is working fine as example 3622 after the reboot it is crashing same as with the intel network adapter
with 1.03 all is running perfekt

**Comment by nemesis122** [comment]
Hi Fabio
Christian has changed some things in the modules now it is working with the gen8 could you have a look i dont know if its only releated to the ixgbe driver 
https://github.com/AuxXxilium/arc-modules/tree/main/broadwellnk-4.4.180
ty

**Comment by nemesis122** [comment]
also this version crashes second boot after install or set back to factory settings

**Comment by fbelavenuto** [comment]
> Hi Fabio Christian has changed some things in the modules now it is working with the gen8 could you have a look i dont know if its only releated to the ixgbe driver https://github.com/AuxXxilium/arc-modules/tree/main/broadwellnk-4.4.180 ty

Ok, thanks.

**Comment by fbelavenuto** [comment]
Please update ARPL, reboot, update modules, addons and lkm, rebuild the loader and test it.

**Comment by nemesis122** [comment]
i will let you know thanks anyway :-)

**Comment by nemesis122** [comment]
Hi Fabio :-)
thank you very much for this Masterpiece Loader  it is working in any case restore / switch to other Modell / Fresh install als the CPU Frequencies is working now perfect.

there is a small issue in the cpu info for the CPU clock but this is only cosmetic.
https://xpenology.com/forum/topic/65408-automated-redpill-loader-arpl/?do=findComment&comment=436739

**Comment by fbelavenuto** [comment]
The ixgbe is working?

**Comment by nemesis122** [comment]
Hi Fabio
Yes ixgbe is working i have tested with 👍 
I3 3240 (2 cores)
16Gb RAM 
1xSSD Volume1 
4x6tb Raid0 Volume2
and intel x520 SFP+ <---
